src/types/Patterns.ts

```diff
@@ -73,7 +73,7 @@ export const Patterns = {
       /^[^ @=<>*\-+#?%&_:;]([a-zA-Z0-9 !?@&#'"()[\]_\-+=<>/*.,;:%\r\n]){9,1999}$/,
     longDescriptionDE:
       /^[^ @=<>*\-+#?%&_:;]([a-zA-ZÀ-ÿ0-9 !?@&#'"()[\]_\-+=<>/*.,;:%\r\n]){9,1999}$/,
-    phone: /^\+(\d{2})+(\(\s\d{3}\))?\s?\d{9,20}$/,
+    phone: /^\+(\d{2})?(\s?)+(\(\d{3}\))?\s?\d{10,13}$/,
```

Comment:
Three things here:
1. fix the sonar bug - something like changing `(\s?)+` to `\s+` should work
2. add a convenience method `isPhone` for that. Scroll down to line 90+ and add it similar to the existing methods.
3. regular expressions are among the things where unit tests make the most sense. Especially if they are not self explanatory like this one here. Someone before added this regex and forgot to add tests. Go to `Patterns.test.ts` and add values for valid and invalid phone numbers similar to the existing tests. Then run `yarn test Patterns` to check if to regular expression works correctly.